fbpic/boundaries/boundary_communicator.py

```diff
@@ -142,18 +164,22 @@ def __init__( self, Nz, Nr, n_guard, Nm, boundaries,
                 # with an infinite order stencil. This would give wrong results
                 if self.size != 1:
                     raise ValueError('Non-local, infinite order stencil \
-                        selected, while performing parallel (MPI) computation.')
+                        selected, while performing parallel computation.')
             else:
                 # Automatic calculation of the guard region size,
                 # depending on the stencil order (n_order)
-                print("Automatic n_guard not supported in this version, " \
-		            "n_guard will be set to n_order * 2 or " \
-		            "will be set to 0 if running in single proc mode.")
```

Comment:
The calculation of `real_kz` and `kz` could be hidden from here, and grouped inside a function 
```python
get_stencil_reach( dz, Nz, n_order )
```
which would be defined inside `utility_methods.py` and would itself calculate `kz` and `kz_real`.